app/controllers/events/list.js

```diff
@@ -28,7 +28,8 @@ export default Controller.extend({
       disableFiltering : true
     },
     {
-      propertyName     : 'speakers.length',
+      propertyName     : 'speakersByState',
```

Comment:
where is speakersByState defined?

Reply:
Nowhere, I didn't know what did the `propertyName` signify, so I was just trying out names during the testing and has been left here. should I keep it as `speakers.length` as it was before? @CosmicCoder96 .

Reply:
yes, please. Thank you.

Reply:
@CosmicCoder96 Done. Please review